test/plugins/knex.spec.js

```diff
@@ -5,6 +5,12 @@
 const agent = require('./agent')
 const plugin = require('../../src/plugins/knex')
 
+const sort = spans => spans.sort((a, b) => a.start.toString() >= b.start.toString() ? 1 : -1)
+
+function normalizeDBStatement (statement) {
```

Comment:
Everything looks great to me. However, are the statements produced through the tests different from those in real world situations? Was just wondering if this normalize function should be in the instrumentation instead.

Reply:
Older versions of knex use double quotes to quote things in queries while the newer ones use the backtick. This normalizes everything to one format so tests don't have to be too verbose and have multiple checks for multiple versions.

Reply:
So it is very specific to the tests. We don't want to modify the query in instrumentation code.